dora/core/common/src/main/java/alluxio/wire/WorkerIdentity.java

```diff
@@ -28,15 +53,19 @@
  * representation and an instance of this class.
  */
 @Immutable
-public final class WorkerIdentity {
+@JsonAdapter(WorkerIdentity.GsonSerde.class)
+@JsonDeserialize(converter = WorkerIdentity.JacksonDeserializeConverter.class)
+@JsonSerialize(using = WorkerIdentity.JacksonSerializer.class)
+public final class WorkerIdentity implements Serializable {
```

Comment:
so there are 3 ser/de : jackson / gson / and java Serializable?
that's too much just for the purpose of ser/de, as WorkerIdentity is only used for membership module only, we dont need to consider anything else including master registration. if there's no usage, can u leave only the gson ser/de plz?

Reply:
Actually there are four: protobuf additionally.

Jackson is used to ser/de into/from `alluxio.wire.WorkerInfo` into JSON.
Gson is added recently to work with etcd.
java Serializable is implemented by `alluxio.wire.WorkerInfo`, which means anything embedded in it should also implement it.
Protobuf is used to ser/de into/from `alluxio.grpc.WorkerInfo`.

Reply:
did you check in EE side? @dbw9580

Reply:
any other additional ser/de frameworks on that side?

Reply:
one used for OPA de/serialize

Reply:
is `WorkerInfo` or `WorkerIdentity` used in those code? At a first glance it does not seem to be related.